generators/client/templates/src/main/webapp/app/services/auth/_auth.jwt.service.js

```diff
@@ -29,13 +29,45 @@
         }
 
         function login (credentials) {
-            var data = {
+            <% if(authenticationType === 'uaa') { %>
+              var data = {
+                username: credentials.username,
+                password: credentials.password,
+                grant_type: "password"
+              };
+              var headers = {
+                'Content-Type': 'application/x-www-form-urlencoded',
+                "Authorization" : "Basic d2ViX2FwcDo="
```

Comment:
you are right, hardcode this makes it unflexible for the future....but encode this each request makes a great overhead. together with a clientDetailsService this should be the first enhancement on this PR.

i had hardcoded it just to make the very plain authorization workin 1:1 as the JJWT do.